### PR TITLE
Feature chatting

### PR DESCRIPTION
**Pull request by Timilof**
basic writing to and reading from db implemented.

### CODE REVIEW

**Comment by Timilof** [comment]
Not yet to be merged or reviewed!!!!

File: helpers/setMessage.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// const db = require('../database/db')
+const User = require('../database/models/user')
+const mongo = require("mongodb");
+const { MongoClient } = require("mongodb");    
+
+module.exports = async (data, collection) => {
+        const uri = process.env.DB_URI;
+
```
Comment (remcopasma) [inline helpers/setMessage.js:8]: Remove white line?

File: static/css/chat.css [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+.send{
+    padding: 14px 10px;
+    border-radius: 8px;
+    border: none;
+    background-color: #fa1c63;
+    color: #fff;
+    letter-spacing: 1px;
+}
+
+
```
Comment (remcopasma) [inline static/css/chat.css:292]: remove whitline

File: static/css/chat.css [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 8px;
+}
+
+.match-name{
+    color: #464646;
+    margin: 10px 0;
+}
+
+
```
Comment (remcopasma) [inline static/css/chat.css:308]: remove whitline

File: helpers/getFromDb.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+  try {
+    await client.connect()
+    const db = client.db(process.env.DB_NAME)
+    let data
+    if (findThis) {
+      data = await db
+        .collection(`${collection}`)
```
Comment (LaupWing) [inline helpers/getFromDb.js:16]: No need to use template strings when it is only one variable

File: helpers/getFromDb.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        .find({ roomID: findThis })
+        .toArray()
+    } else {
+      data = await db
+        .collection(`${collection}`)
```
Comment (LaupWing) [inline helpers/getFromDb.js:21]: Same as above :D

File: helpers/setMessage.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  const client = new MongoClient(uri, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+
+  try {
+    await client.connect()
+    const db = client.db(process.env.DB_NAME)
+    const fullDump = await db.collection(`${collection}`).insertOne({
```
Comment (LaupWing) [inline helpers/setMessage.js:16]: No nee dot use template strings !

File: helpers/updateLastSent.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+  const client = new mongo.MongoClient(uri, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  })
+  try {
+    await client.connect()
+    const db = client.db(process.env.DB_NAME)
+    const updatedDocument = await db
+      .collection(`${collection}`)
```
Comment (LaupWing) [inline helpers/updateLastSent.js:14]: NO need to use template strings

File: views/pages/chat.ejs [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+        <% if (partner) { %>
+        <img class="match-image chat-img" src="<%=partner.img%>" alt="" />
+        <h3 class="match-name chat-name"><%=partner.name%></h3>
+        <a href="/unmatch/<%=partner.id%>" class="unmatch-heart smaller"
+          >unmatch <%=partner.name%></a
+        >
+        <% } %>
+      </div>
+      <ul class="chat-container">
+        <% if (user) { %> <% if (chatHistory) {
```
Comment (LaupWing) [inline views/pages/chat.ejs:21]: Indendt and next line for more readability

**Pull request merged**